### ISSUE TITLE
VM6404HB Matrix Support

### ISSUE THREAD

**Issue by ukiews** [bitfocus/companion-module-aten-matrix#12]
Would this work with the VM6404HB:
https://www.aten.com/us/en/products/professional-audiovideo/video-matrix-switches/vm6404hb/
?

**Comment by NoahCallaway** [comment]
Yes it should work, although the module hasn't been tested with that specific model. 

https://github.com/bitfocus/companion-module-aten-matrix/blob/master/companion/HELP.md
